### ISSUE TITLE
Not able to copy to /usr/bin and update cache give me error of kexts are unsigned

### ISSUE THREAD

**Issue by harveson** [david-cako/T430-Hackintosh#4]
Did you disable the system integrity protection?

**Comment by david-cako** [comment]
I'm fairly certain I didn't, I will check tomorrow and let you know.

\- David

**Comment by harveson** [comment]
By the way, I saw you were on 10.11.5. The only version I can get is 10.11.6. I guess that might be the reason for the problem? Also, can you give me the version of Unibeast/Clover/KextWizard that you used? Thanks!

**Comment by david-cako** [comment]
SIP *is* disabled.  Sorry about that.

Clover version is 3577, Unibeast and KextWizard are a don't-know, but KextWizard doesn't appear to have been updated in a few years (at least on softpedia), so I'm gonna say 99% sure it was 3.7.11.

**Comment by harveson** [comment]
Thank you for replying. 
I could not disable SIP during post installation. I found an article (https://www.tonymacx86.com/threads/explaining-os-x-el-capitan-security-changes-workarounds-and-current-information.170611/) suggesting adding the following to the config.plist before the smcbios setting.
I will let you know if it works after I add these lines.

<key>RtVariables</key>
    <dict>
        <key>CsrActiveConfig</key>
        <string>0x3</string>
        <key>BooterConfig</key>
        <string>0x28</string>
    </dict>

**Comment by david-cako** [comment]
rootless = 0 didn’t do it?  I’m pretty sure that’s how I did it.

**Comment by harveson** [comment]
"Early OS X 10.11 El Capitan betas contained a new boot flag to disable rootless security called rootless=0. This has been removed in recent beta builds, and replaced with NVRAM csr-active-config." . I guess that is why you did not have problems as you were installing earlier El Capitan builds (however they are not available to download anymore).

**Comment by david-cako** [comment]
Ah, that makes sense!  Sounds like the NVRAM csr is the way to go.

**Comment by harveson** [comment]
Thanks for providing the repo. It really helps. How's the usb wifi working? Does it have problem with sleeping?

**Comment by david-cako** [comment]
Sleeping does not work at all (it just turns off the displays I think?), as is usually the case with hackintosh setups.

The USB wifi is about what I would expect.  That particular edimax model will only do ~20mbps, so I use [this D-Link](https://www.amazon.com/D-Link-Wireless-Network-Adapter-DWA-171/dp/B00CL58JYQ/) at work.  It works really well, and provides proper n/ac speeds.

They both use the same base driver, though, and I am able to hot-swap them.  I keep the D-Link plugged into a hub, and use the Edimax when my computer leaves my desk.

**Comment by harveson** [comment]
Could you let me know what dsdt patch you applied?

**Comment by david-cako** [comment]
All of the ones found here.  I don't believe I used any other.  I also seem to remember an issue with the battery DSDT?  Try that that one, if not I used one that was for a similar Thinkpad.

https://www.tonymacx86.com/threads/guide-lenovo-t430-el-capitan.175935/

**Comment by harveson** [comment]
I got my T430 working. My configuration is exactly like yours. 8GB RAM+1600x900+NVS5400. I found my VGA to be working, which is weird. I also found that closing lid does not result in sleep or turning off the internal display, which is annoying for using it with an external monitor. It seems like the system is not picking up the "close lid" signal correctly. 
But anyway, your guide is amazing and saves me a lot of trouble. Really appreciate you provide this guide.
